README.md

```diff
@@ -114,6 +114,21 @@ Spark Version | Release # | Binary Location
 	
 	df.filter(df.airportCode >= 'CAA').select("airportCode",'airportName').save("airportcodemapping_df", "com.cloudant.spark")	    
 	
+In case of doing multiple operations on a dataframe (select, filter etc.)
+you should persist a dataframe. Othewise, every operation on a dataframe will load the same data from Cloudant again.
+Persisting will also speed up computation. This will persist an RDD in memory: 
+
+```
+df.cache() 
```

Comment:
Could you please work this df.cache() statement into the code snippet above? I know it is in the actual CloudantDF.py script but having it in the snippet would be useful, too. Right after we load the RDD, let`s put:

```
// cache RDD in memory
df.cache()

// to cache RDD on disk
// df.persist(storageLevel = StorageLevel(True, True, False, True, 1))
```